docs/contacts/contacts_overview.rst

```diff
@@ -0,0 +1,84 @@
+
+Mautic tracks these Contacts, but usually keeps them hidden so as not to clutter up your Segment.
+  
+Visitors are worth tracking, because these could be future customers. By tracking them before they have any interaction, you can retain a log of when they visited your site, which allows you to get a picture of their activity prior to engaging with you.
+
+You can use the filter in the Contacts screen to display only visitors by using the this command ``is:anonymous`` in the search bar above the Contacts list.
+
+.. image:: images/contacts-anonymous.png
+   :alt: Screenshot of anonymous Contact
+
+**What you see** - Switching to the anonymous Contacts view displays the IP addresses for visitors to Landing Pages tracked with your Mautic tracking code. 
+If you have an IP lookup service configured in **Settings > System Settings > Miscellaneous Settings**, you will also see an approximate location of these Contacts. Mautic uses MaxMind Geolite2 City by default. The location is based on the Contact’s Internet Service Provider, and may not be the exact location of the Contact.
+
+**Individual Contacts** - Click an IP address to display a Contact record, similar to known Contacts. If there’s any information on an anonymous Contact, you can see it here. This data can include:
+
+* Landing Pages the Contact has visited
+
+* Forms submitted
+
+* Scoring
+
+* Data from Forms (which don’t include the unique identifier, since that would make the contact known), etc.
+
+.. note:: 
+
+    * In **Settings > Configuration > Tracking Settings**, you can enable the **Do Not Track 404 error for anonymous Contacts** option to not track page hits on any 404 error page tracked by the tracking code. This option helps prevent bots tracking.
+
+    * Data for anonymous Contacts isn't available for segmentation or reporting. Once the Contact is known, the data is available. This applies to non-campaign based dynamic web content filters.
+
+
+.. vale off
+
+Standard Contacts
+-----------------
+
+.. vale on
+
+The second type of Contact is a standard - or known - Contact. These Contacts have identified themselves via a Form or other source. You may also have more information about them from previous interactions, or from a third-party system such as a Customer Relationship Management (CRM) tool. 
+As a result, these Contacts typically have a name, Email, and other identifying information which can be associated with the Contact.
+
+The standard Contact is the preferred Contact within Mautic. These are Contacts which may have started as a visitor, but at some point provided additional information such as a name, Email address, social network handle, or other identifying characteristics which have enabled you to connect up the activity on your website with a known person. You can nurture these Contacts through the Mautic marketing automation platform, learn more about their behaviour, and take specific actions as a result of this information.
```

Comment:
What do we mean by 'the standard Contact'? I'm not clear how this is different to what we stated earlier.

Reply:
It's what we have in the old docs and it's more like suggesting to the user that it is the preferred contact in Mautic. This sentence can also be removed. It's not necessary tho

Reply:
I would remove that sentence personally - I don't think that it adds anything to the text.

So I'd say something like:

These are visitors which may have started as anonymous, but at some point provided additional information such as a name, Email address, social network handle, or other identifying characteristics which have allowed you to connect up the activity on your website with a known person.

In the sentence before (line 61) I would also remove the reference to 'standard':

'The second type of Contact is a known Contact. These Contacts have identified themselves via a Form or other source. You may also have more information about them from previous interactions, or from a third-party system such as a Customer Relationship Management (CRM) tool.